src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/WorkingDayCountService.kt

```diff
@@ -24,6 +25,10 @@ class WorkingDayCountService(
     return from.getDaysUntilInclusive(to).filter { it.isWorkingDay(bankHolidays) }.size
   }
 
+  fun getDifferenceInWorkingDays(from: LocalDate, to: LocalDate): Int {
+    return from.getDaysUntilExclusiveEnd(to).filter { it.isWorkingDay(bankHolidays) }.size
+  }
```

Comment:
Is getDifferenceInWorkingDays only ever going to be used for today's date? If yes, maybe we change it to reflect that and call it something like:

getCompleteWorkingDaysFromNowUntil(to: LocalDate)?

I think it would be clearer what it's doing then? My concern is that it's not obvious what difference is between the two functions we have.

Reply:
Or could always call it getCompleteWorkingDatesCount() if you want to pass in both arguments? and in this case, update 'from' to be 'fromExclusive' and 'to' to be 'toInclusive'? Then update argument names on other function use same pattern?

Reply:
yes I agree that adding the term "complete" to the function name will probably help explain the purpose of this function which rounds down the count of days remaining. Personally I'm not sure the `fromExclusive` and `toInclusive` arg names would help my understanding...

Reply:
Good call, I've gone with `getCompleteWorkingDaysFromNowUntil` 👍